apps/gruenbuch/models.py

```diff
@@ -20,7 +20,8 @@ class GruenbuchOverviewPage(MetadataPageMixin, Page):
         blank=True,
         on_delete=models.SET_NULL,
         related_name='+',
-        help_text='Image ratio has to be 2:1 or 4:3.'
+        help_text='Image ratio has to be 2:1 or 4:3. Intro image is not '
+                  'shown for Easy Language (Leichte Sprache).'
```

Comment:
Since we had to add translations to the wagtail for the categories ect. i discussed with Max and we will do it for all strings in admin so we don't need the 'Leichte Sprache' sorry!